mediacapture-streams/stream-api/mediastream/mediastream-finished-add.html

```diff
@@ -1,49 +0,0 @@
-<!doctype html>
-<html>
-<head>
-<title>Adding a track to a finished MediaStream</title>
```

Comment:
Do you mean "adding a track to an inactive MediaStream"? If yes, I think it is acceptable.

For this test, sounds like changing the assertion in line 36 is able to do this.